doc/pod1/cellcc_config.pod

```diff
@@ -103,12 +103,40 @@ mysql database called "cellcc" on host db.example.com using port 3306.
 
 The username and password to use when connecting to the B<db/ro/dsn> database.
 
+=item B<db/ro/options>
+
+This optional directive can be used to add / alter the settings to be used when
+contacting the database. For example, the admin can specify the timeout to use
+when reading or connecting to the B<db/ro/dsn> database. For the MySQL driver,
+the following timeout options are available:
+
+- mysql_connect_timeout
+- mysql_read_timeout
+- mysql_write_timeout
+
+In the following example, requests to the server will timeout after 5 seconds:
```

Comment:
specifically, this sets the timeout for connecting and for read requests to 5 seconds, but leaves write requests at the default timeout of 5 minutes, yes? we don't need to go into the nuances of the different timeout options, but we should at least be correct here (or just use an easier-to-explain example while still being correct)